exchange/utils_test.go

```diff
@@ -468,6 +470,12 @@ func TestExtractBidderExts(t *testing.T) {
 			givenImpExtPrebidBidders: map[string]json.RawMessage{"bidderA": bidderAJSON},
 			expected:                 map[string]json.RawMessage{"bidderA": bidderAJSON},
 		},
+		{
+			description:              "imp.ext.tid",
+			givenImpExt:              map[string]json.RawMessage{"bidderA": json.RawMessage(`{"shouldBe":"Ignored"}}`), "tid": json.RawMessage(`{`)},
+			givenImpExtPrebidBidders: map[string]json.RawMessage{"bidderA": bidderAJSON},
+			expected:                 map[string]json.RawMessage{"bidderA": bidderAJSON},
+		},
```

Comment:
I recommend to remove this test case and add tid to the existing "Special Names Ignored - imp.ext.BIDDER" case.

Reply:
done